tests/aws/services/opensearch/test_opensearch.py

```diff
@@ -360,13 +358,8 @@ def test_create_domain(self, opensearch_wait_for_cluster, aws_client):
             aws_client.opensearch.delete_domain(DomainName=domain_name)
 
     @markers.aws.only_localstack
-    @pytest.mark.parametrize(
-        "engine_version",
-        # Test once per major version
-        ["OpenSearch_1.3", "OpenSearch_2.5"],
```

Comment:
I guess it is good enough to test with the latest version?

Reply:
Yep, that is the intention. It is by far the version which is used most often, and it turned out that the test actually failed with the latest version (it was an issue in the test though, since the security plugin got more strict on the passwords).